packages/cli/src/util/colorizer.ts

```diff
@@ -0,0 +1,23 @@
+import { mapValues, isArray } from 'lodash';
+import { Dictionary } from '@stoplight/types';
+
+export type ValuesTransformer = (values: Dictionary<unknown>) => Dictionary<string | string[]>;
+
+export const PRE_PARAM_VALUE_TAG = '~pre~';
+export const POST_PARAM_VALUE_TAG = '~post~';
+
+const taggedParamsValues = new RegExp(`(${PRE_PARAM_VALUE_TAG})(.*?)(${POST_PARAM_VALUE_TAG})`, 'gm');
```

Comment:
```suggestion
const taggedParamsValues = new RegExp(`${PRE_PARAM_VALUE_TAG}(.*?)${POST_PARAM_VALUE_TAG}`, 'gm');
```

The capture groups around tags are not needed.